Makefile

```diff
@@ -89,7 +89,7 @@ $(BUILD_DIR):
 
 
 $(BUILD_DIR)/ipmanager: $(BUILD_DIR) $(VENDOR_DIR)
-	$(DOCKER_EXEC) bash -xc 'go build --ldflags "-extldflags \"-static\"" \
+	$(DOCKER_EXEC) bash -xc 'apt-get update; apt-get install -y libpcap-dev; go build --ldflags "-extldflags \"-static\"" \
```

Comment:
Also this patch breaks non-containeraized builds due to insufficient permissions for running `apt-get`